src/main/java/com/microsoft/sqlserver/jdbc/IOBuffer.java

```diff
@@ -5,28 +5,17 @@
 
 package com.microsoft.sqlserver.jdbc;
 
-import java.io.ByteArrayInputStream;
-import java.io.ByteArrayOutputStream;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
-import java.io.Reader;
-import java.io.Serializable;
-import java.io.UnsupportedEncodingException;
+import com.microsoft.sqlserver.jdbc.SQLServerConnection.FedAuthTokenCommand;
+import com.microsoft.sqlserver.jdbc.dataclassification.SensitivityClassification;
+
+import javax.net.SocketFactory;
+import javax.net.ssl.*;
+import java.io.*;
```

Comment:
Please use individual class imports as was here previously, instead of * imports.

Reply:
in multiple places, our coding standard requires imports only for packages that's used, please remove all these type of changes